lib/linter.js

```diff
@@ -262,8 +262,8 @@ class Linter {
     return currentSource;
   }
 
-  async updateTodo(linterOptions, results) {
-    await this._todoHandler.update(linterOptions.filePath, results);
+  async updateTodo(linterOptions, results, daysToDecay) {
+    await this._todoHandler.update(linterOptions.filePath, results, daysToDecay);
```

Comment:
Can we call this more generally `todoConfig`? Its _possible_ that we add more config values other than the decay days itself...

Reply:
Ah yes, this was a spot I missed. I'll update.

Reply:
Done.